exchanges/okx/okx_types.go

```diff
@@ -1670,30 +1670,30 @@ type RFQOrderLeg struct {
 
 // CreateRFQInput RFQ create method input.
 type CreateRFQInput struct {
-	Anonymous           bool          `json:"anonymous"`
-	CounterParties      []string      `json:"counterparties"`
-	ClientSuppliedRFQID string        `json:"clRfqId"`
-	Legs                []RFQOrderLeg `json:"legs"`
+	Anonymous      bool          `json:"anonymous"`
+	CounterParties []string      `json:"counterparties"`
+	ClientRFQID    string        `json:"clRfqId"`
+	Legs           []RFQOrderLeg `json:"legs"`
 }
 
 // CancelRFQRequestParam represents cancel RFQ order request params
 type CancelRFQRequestParam struct {
-	RfqID               string `json:"rfqId"`
-	ClientSuppliedRFQID string `json:"clRfqId"`
+	RfqID       string `json:"rfqId"`
+	ClientRFQID string `json:"clRfqId"`
 }
 
 // CancelRFQRequestsParam represents cancel multiple RFQ orders request params
 type CancelRFQRequestsParam struct {
-	RfqID               []string `json:"rfqIds"`
-	ClientSuppliedRFQID []string `json:"clRfqIds"`
+	RfqID       []string `json:"rfqIds"`
+	ClientRFQID []string `json:"clRfqIds"`
 }
 
 // CancelRFQResponse represents cancel RFQ orders response
 type CancelRFQResponse struct {
-	RfqID               string `json:"rfqId"`
-	ClientSuppliedRfqID string `json:"clRfqId"`
-	StatusCode          string `json:"sCode"`
-	StatusMsg           string `json:"sMsg"`
+	RfqID       string `json:"rfqId"`
+	ClientRfqID string `json:"clRfqId"`
```

Comment:
I am being pedantic but some are `ClientRfqID` and `ClientRFQID` could we please standardise?

Reply:
Well... You see ... If we're going to be Pedantic about it then prepositions should be lowercase ...
So RfQId 😂

Reply:
I'll go for RfqID
I tried all three and it feels best.

This highlights that 1688 is already wrong too. That's an array.

Reply:
Fixed in 3019f93b